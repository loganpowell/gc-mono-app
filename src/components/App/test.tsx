// import renderer from "react-test-renderer";
// import { IntlProvider } from "react-intl";
// import English from "@lang/en.json";

// import App from ".";

// describe("<App />", () => {
//   const component = (props) =>
//     renderer.create(
//       <IntlProvider locale={"en"} messages={English}>
//         <App {...props} />
//       </IntlProvider>,
//     );

//   it("renders correctly", () => {
//     expect(component({}).toJSON()).toMatchSnapshot();
//   });
// });
