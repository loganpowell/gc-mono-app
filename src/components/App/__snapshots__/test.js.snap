// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="links"
  >
    <ul>
      <li>
        <a>
          Your Uploads
        </a>
      </li>
      <li />
    </ul>
  </div>
  <div
    className="logo"
  >
    <img
      alt="gaza care logo"
      src="test-file-stub"
    />
  </div>
  <div
    className="app-info"
  >
    find first aid essentials to help those injured in war zones
  </div>
  <div
    className="field"
  >
    <div
      className="control"
    >
      <input
        className="input search-bar"
        onChange={[Function]}
        placeholder="search"
        type="text"
      />
    </div>
  </div>
  <div
    className="field"
  >
    <div
      className="control search-action"
    >
      <button
        className="button is-primary search-button"
        onClick={[Function]}
      >
        search
      </button>
    </div>
  </div>
  <div
    className="videos"
  />
</div>
`;
